packages/components/src/Input/Input/_docs/Input.stickersheet.stories.tsx

```diff
@@ -0,0 +1,125 @@
+import React from "react"
+import { Meta } from "@storybook/react"
+import { MeatballsIcon, WritingIcon } from "~components/Icons"
+import {
+  StickerSheet,
+  StickerSheetStory,
+} from "~storybook/components/StickerSheet"
+import { Input, InputProps } from "../index"
+import { InputStatus, InputTypes } from "../types"
+
+export default {
+  title: "KAIO-staging/Inputs/Input",
+  parameters: {
+    chromatic: { disable: false },
+    controls: { disable: true },
+    a11y: {
+      config: {
+        rules: [
+          {
+            // Built with no label on purpose, to be used within `TextField` where label is present
+            id: "label",
+            enabled: false,
+          },
+        ],
+      },
+    },
+  },
+} satisfies Meta
+
+const InputExampleGroup = (props: InputProps): JSX.Element => (
+  <div className="grid gap-16">
+    {InputTypes.map(type => (
+      <Input
+        key={type}
+        value={type}
+        type={type}
+        startIconAdornment={
+          <WritingIcon
+            role="presentation"
+            data-sb-a11y-color-contrast-disable={
+              props.disabled ? "true" : "false"
+            }
+          />
+        }
+        endIconAdornment={
+          <MeatballsIcon
```

Comment:
I'm wondering if we should change out the icon on the success stories to the tick svg? Otherwise there's no discernible difference from default and success states (though you could certainly argue that that would be be the point)

Reply:
We could potentially add a line and Canvas to provide more guidance in the MDX if we want to keep our chromatic tests with the same icons 🤔

Reply:
Not worth it for these stickersheets. Good callout that the styling isn't discernible enough, but we should consider whether or not to enforce a success styling to begin with, cause the design docs don't even mention "success" patterns. Which I'm assuming because forms tend to move on after being successful, and in other cases, we want errors to stand out. So I'd question the necessity of this variant anyway 😅

Reply:
We seem to have a few of these across our form components that don't change styles. I would guess that the "success" variant changed the border to green in the past

Reply:
Logged :D https://cultureamp.atlassian.net/browse/KDS-1821